### PR TITLE
add zuul config

### PR DESCRIPTION
**Pull request by TomasTomecek**
EDIT (jpopelka): https://zuul-ci.org/docs/zuul/user/config.html#project
EDIT2 (jpopelka): http://post-office.corp.redhat.com/archives/user-cont/2019-July/msg00086.html

### CODE REVIEW

File: .zuul.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+- project:
+    name: packit-service/ogr
+    check:
+      jobs:
+        - noop
```
Comment (jpopelka) [inline .zuul.yaml]: How is this going to help us?

Comment (TomasTomecek) [reply .zuul.yaml]: see the email, this is the first step to test the connection b/w ogr and sf.io's zuul

**Comment by morucci** [comment]
recheck

**Comment by TomasTomecek** [comment]
all green, ready for review

**Comment by TomasTomecek** [comment]
that's a good question; I don't think we have set up testing of master branch, let me see

**Comment by TomasTomecek** [comment]
upstream zuul docs: https://zuul-ci.org/docs/zuul/user/badges.html

merging for now

**Comment by lachmanfrantisek** [comment]
> merging for now

No rebase..;-( You are making me sad...

**Pull request merged**